src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -467,7 +471,7 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
 {1} is the exception ToString() result</comment>
   </data>
   <data name="Log_LocalSourceNotExist" xml:space="preserve">
-    <value>The local source '{0}' doesn't exist.</value>
+    <value>The local source or fallback folder '{0}' doesn't exist. Create directory, or fix packageSource/fallbackPackageFolder declarations.</value>
```

Comment:
Create _the_ directory or fix ...

Remove the comma.

Reply:
fixed in next iteration.